app/assets/stylesheets/active_admin/_forms.scss

```diff
@@ -1,35 +1,65 @@
 //= depend_on_asset "active_admin/datepicker/datepicker-input-icon.png"
-// -------------------------------------- Active Admin Forms
-form {
-  /* Reset margins & Padding */
-  ul, ol, li, fieldset, legend, input, textarea, select, p { margin:0; padding:0; }
-  ol, ul { list-style: none }
+
+input[type=text],
+input[type=password],
+input[type=email],
+input[type=number],
+input[type=url],
+input[type=tel],
+textarea {
+  border: $border-width solid #c9d0d6;
+  border-radius: 3px;
+  outline: none;
+  padding: 6px 8px;
+  width: calc(80% - #{$text-input-total-padding});
+
+  &:focus {
+    border: $border-width solid #99a2aa;
```

Comment:
Color literals like `#99a2aa` should only be used in variable declarations; they should be referred to via variable everywhere else.